docs/en/middleware.rst

```diff
@@ -201,25 +201,95 @@ Both redirect handlers share the same configuration options:
 
 For example::
 
+    use Authorization\Exception\MissingIdentityException;
+    
     $middlewareQueue->add(new AuthorizationMiddleware($this, [
         'unauthorizedHandler' => [
             'className' => 'Authorization.Redirect',
-            'url' => '/users/login',
+            'url' => '/pages/unauthorized',
             'queryParam' => 'redirectUrl',
             'exceptions' => [
                 MissingIdentityException::class,
                 OtherException::class,
             ],
         ],
     ]));
+    
+All handlers get the thrown exception object given as a parameter.
+This exception will always be an instance of ``Authorization\Exception\Exception``.
+In this example the ``Authorization.Redirect`` handler just gives you the option to 
+specify which exceptions you want to listen to.
+
+So in this example where we use the ``Authorization.Redirect`` handler we can
+add other ``Authorization\Exception\Exception`` based exceptions to the 
+``execeptions`` array if we want to handle them gracefully::
 
-You can also add your own handler. Handlers should implement
-``Authorization\Middleware\UnauthorizedHandler\HandlerInterface``, be suffixed
-with ``Handler`` suffix and reside under your app's or plugin's
-``Middleware\UnauthorizedHandler`` namespace.
+    'exceptions' => [
+        MissingIdentityException::class,
+        ForbiddenException::class
+    ],
+
+See the `RedirectHandler source <https://github.com/cakephp/authorization/blob/master/src/Middleware/UnauthorizedHandler/RedirectHandler.php>`__
```

Comment:
Anyone else think that linking to `master` could cause problems down the road once it will be used for a new major version?

Reply:
I thought linking to an example in the plugin would be better than copying the source.

Reply:
Sure, my only concern is that the linked code can change to the point of being incompatible, given that it's the master branch. Maybe we should link to tags, or to `x.next` branches? 🤔

Ideally there would be a URL format that would allow linking to the latest tag of a major, but I don't think GitHub supports that.

Reply:
Sure, I'll do that